invisible_cities/cities/icaro.py

```diff
@@ -0,0 +1,532 @@
+"""
+-----------------------------------------------------------------------
+                                Icaro
+-----------------------------------------------------------------------
+
+
+"""
+
+import numpy as np
+import pandas as pd
+import tables as tb
+
+from .. reco                import tbl_functions as tbl
+from .. reco.corrections    import ASectorMap
+from .. reco.corrections    import maps_coefficient_getter
+from .. reco.corrections    import correct_geometry_
+from .. reco.corrections    import norm_strategy
+from .. reco.corrections    import get_normalization_factor
+from .. reco.corrections    import apply_all_correction
+from .. io.krmaps_io        import write_krmap
+from .. io.krmaps_io        import write_krevol
+from .. io.krmaps_io        import write_mapinfo
+from .. core                import fit_functions as fitf
+from .. core.core_functions import in_range
+from .. core.core_functions import shift_to_bin_centers
+
+from .. dataflow          import dataflow as fl
+
+from .. types.ic_types    import Measurement
+from .. types.ic_types    import MasksContainer
+
+from .  components import city
+from .  components import print_numberofevents
```

Comment:
`print_numberofevents`, `quality_check` and `kr_selection` are not defined in `components.py`, so the code gives `ImportError`

Reply:
I left the rest of the city as it was before my commits, I only included the imports that I needed for my part and the `compute_map` function, but did not modify anything else, so it should be OK when the different parts of the code are included... Right now here it's the skeleton that Brais drafted and some of those imports are not done yet (It's not that they are not in `components.py` because they are somewhere else, I guess they are nowhere right now).